py/server/tests/test_udf_return_java_values.py

```diff
@@ -290,9 +297,30 @@ def test_np_ufunc(self):
         @numba.vectorize([numba.float64(numba.float64)], nopython=True)
         def nbsin(x):
             return np.sin(x)
+
         t3 = empty_table(10).update(["X3 = nbsin(i)"])
         self.assertEqual(t3.columns[0].data_type, dtypes.double)
 
+    def test_md_np_array(self):
+        np_1d_array = np.array([1, 2, 3, 4, 5, 6, 7, 8], dtype=np.int64)
+        np_2d_array = np_1d_array.reshape((4, 2))
+        np_3d_array = np_1d_array.reshape((2, 2, 2))
+
+        def f(ndim):
+            if ndim == 1:
+                return np_1d_array
+            elif ndim == 2:
+                return np_2d_array
+            elif ndim == 3:
+                return np_3d_array
+            else:
+                raise ValueError("ndim must be 1, 2, or 3")
+
+        for ndim in [1, 2, 3]:
```

Comment:
There should also be an equivalent set of 3 tests with type hints to make sure that our return hints are working properly.

Reply:
For the type hinted version, the cast in the query string should not be needed.